segment.py

```diff
@@ -13,33 +11,37 @@ def lambda_handler(event, context):
     # Get the object from the event and show its content type
     # This job is triggered by FT_VideoConversions
 
-    Bucket = event[0]['dynamodb']['NewImage']['Bucket']
-    Key = event[0]['dynamodb']['NewImage']['Key']
-    ConversionID = event[0]['dynamodb']['NewImage']['ConversionID']
-    QueueMessageID = event[0]['dynamodb']['NewImage']['QueueMessageID']
+    Row = event[0]['dynamodb']['NewImage']
+    Bucket = Row['Bucket']
+    Filename = Row['Filename']
+    Path = Row['Path']
+    ConversionID = Row['ConversionID']
+    QueueMessageID = Row['QueueMessageID']
```

Comment:
Not part of your code here, but We should prob consider renaming this to StatusQueueMessageID for clarity.